sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1842,6 +1844,8 @@ export interface SharePropertiesInternal {
     // (undocumented)
     enabledProtocols?: string;
     // (undocumented)
```

Comment:
comment?

Reply:
This interface is auto-generated, there's no comments in swagger spec, so no comments in auto-generated source code.  Won't want to modify auto-generated code every time.